src/common/components/Toggle/Toggle.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react'
+import cn from 'classnames'
+
+export type TToggleProps = {
+  classNames?: string
+  selectedClass?: string
+  toggleHandler: (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => void
+}
+
```

Comment:
@sanjanadhikari1 I think we need two props are reqired
1. selected/checked -> boolean 
2. toogleHandler -> selected flag should pass

Without these, how to pass selected flag to pass `toggle` component?
How to know On/Off state from togglehandler func?